src/index.js

```diff
@@ -0,0 +1,125 @@
+/* global THREE, AFRAME */
+const Vimeo = require('./../ext/vimeo-threejs-player/dist/vimeo-threejs-player.min')
+
+AFRAME.registerComponent('vimeo', {
+  multiple: true,
+  schema: {
+    id: { type: 'number', default: 0 },
+    autoplay: { type: 'bool', default: true },
+    autoload: { type: 'bool', default: true },
+    muted: { type: 'bool', default: false },
+    loop: { type: 'bool', default: true },
+    volume: { type: 'number', default: 1.0 },
+    leftEye: { type: 'selector' },
+    rightEye: { type: 'selector' },
+    quality: { type: 'string', default: 'auto' }
+  },
+  init: function () {
+    console.log('Hello Vimeo 🎥 version: ', Vimeo.version)
+
+    if (this.data.leftEye && this.data.rightEye) {
+      console.log('[Vimeo] loading stereo spherical video 🌐')
```

Comment:
Do we want to always console logging in the core component?

Reply:
https://github.com/vimeo/aframe-vimeo-component/pull/10/commits/2438354729bcea92840ba716e9fd89d4d45c7099